opm/grid/CpGrid.hpp

```diff
@@ -998,7 +1001,7 @@ namespace Dune
         double cellCenterDepth(int cell_index) const;
 
 
-        const Vector faceCenterEcl(int cell_index, int face) const;
+        const Vector faceCenterEcl(int cell_index, int face, const Dune::cpgrid::Intersection& intersection) const;
```

Comment:
So, my impression is that we cannot get rid of cell_index and face, since they are either 
cell_index and face from the neighboring cell inside, or 
cell_index and face from the neighboring cell outside. 

I could be wrong though

Reply:
![Screenshot from 2024-03-19 14-45-23](https://github.com/OPM/opm-grid/assets/89510440/46d33906-2844-47c1-9b57-8caf363bc995)

Reply:
(that piece of code belongs to Transmisibility_impl.hpp, change made in OPM/opm-simulators#5218)